tests/cases/Issue89.ts

```diff
@@ -0,0 +1,4 @@
+export class ^FileLogger ^extends /**/^One ^implements/**/ ^IDisposable{
+	^	
+}
+^
```

Comment:
Also add a marker inside the `/**/` comment block to make sure they are parsed as comment